### PR TITLE
added missing headers

### PR DESCRIPTION
**Pull request by looooo**

### CODE REVIEW

**Comment by jjhelmus** [comment]
This feedstock needs to be re-rendered to pick up some changes.  To do this

* Install conda-smithy if not already installed: `conda install -c conda-forge conda-smithy`
* In the root tk-feedstock directory run `conda smithy rerender`
* check in the files which were changed and make a new commit.

**Comment by jjhelmus** [comment]
LGTM, thanks for putting this together @looooo

File: appveyor.yml [hunk 1]
```diff
@@ -20,28 +20,8 @@ environment:
...
-      VC_VERSION: 10
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      VC_VERSION: 14
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      VC_VERSION: 14
-      CONDA_PY: 35
```
Comment (jakirkham) [inline appveyor.yml]: Umm...this isn't good.

cc @pelson

**Comment by jakirkham** [comment]
We are going to need to pull the Windows packages from this build. Also need to re-render and rebuild to fix Windows.

File: appveyor.yml (hunk 1 above)
Comment (jakirkham) [inline appveyor.yml]: Have marked the Windows packages as broken. Will add a new PR to fix this momentarily.

screen shot 2016-09-15 at 10 49 30

Comment (jakirkham) [inline appveyor.yml]: Please see PR ( https://github.com/conda-forge/tk-feedstock/pull/6 ) for the fix.

**Comment by jjhelmus** [comment]
Sorry about that @jakirkham, I missed to Windows issue in my review.  Thanks for pulling these files and for the fix.  I'll look at it momentarily.

**Comment by jakirkham** [comment]
No worries. We need to figure out a better way to handle re-renderings on feedstocks like this one using `vc` ( https://github.com/SciTools/conda-build-all/issues/65 ).

**Pull request merged**